Mobile menu

Sava12012 opened the pull request.

Sava12012 committed:
1

Sava12012 committed:
Мобільне меню та кнопки

Зверстав модалку для мобільного меню та планшету. Підключив до кнопок js. Кнопки відображаються тільки в планшетному та мобільному режимі. width та hight в модалці вистави 100%, після закінчення оформлення хедера та хіро має стати по ширині вюпорту.

gr18teamlead commented on src/partials/header.html:
```diff
@@ -1,27 +1,14 @@
 <!-- Header section -->
+
 <header class="header">
   <div class="container heaer_container">
```
помилка: не heaer а header. Плюс всі класси через символи '__' а не один '_'

gr18teamlead commented on src/partials/header.html:
```diff
@@ -1,27 +1,14 @@
+
 <header class="header">
   <div class="container heaer_container">
     <a class="header-logo" href="./index.html" aria-label="logo icecream">
```
теж саме, класси. Блок__елемент--модифікатор. "logo" - це єлемент блока "header"

gr18teamlead commented on src/partials/header.html:
```diff
@@ -1,27 +1,14 @@
 <header class="header">
   <div class="container heaer_container">
     <a class="header-logo" href="./index.html" aria-label="logo icecream">
       <svg class="header-icon" width="199" height="50">
```
Теж клас виправти

gr18teamlead commented on src/partials/header.html:
```diff
@@ -1,27 +1,14 @@
-          class="header-icon_svg"
-          href="../src/images/svg/symbol-defs.svg#icon-logo"
-        ></use>
+        <use class="header-icon_svg" href="../src/images/svg/symbol-defs.svg#icon-logo"></use>
```
теж клас

gr18teamlead commented on src/sass/components/_mobile-menu.scss:
```diff
@@ -1,6 +1,65 @@
+  padding-top: 60px;
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-bottom: 185px;
```
Для таблетки у макеті інші падінги

gr18teamlead commented on src/sass/components/_mobile-menu.scss:
```diff
@@ -1,6 +1,65 @@
+}
+
+.mobile-menu_nav {
+  margin-top: 32px;
```
Тут марджин 20 px. бо він рахуеться до полоски а не до слова

gr18teamlead commented on src/sass/components/_mobile-menu.scss:
```diff
@@ -1,6 +1,65 @@
+  margin-top: 32px;
+
+  @media screen and (min-width: 768px) {
+    margin-top: 33px;
```
і тут теж відповідно зміниться

gr18teamlead commented on src/sass/components/_mobile-menu.scss:
```diff
@@ -1,6 +1,65 @@
+  line-height: 1.28;
+  color: white;
+
+  padding-top: 13px;
```
тут краще задати висоту на тег li і потім відцентрувати слова. Бо висота всіх блоків однакова, а падінги в середині у всіх різні. Зробити як ми в домашці робили іконки в соцмережах

gr18teamlead commented on src/sass/components/_mobile-menu.scss:
```diff
@@ -13,6 +72,29 @@
+  color: var(--main-color);
+}
+
+.munu-btn_icon {
```
Кнопка закриття не того розміру. Плюс вона не на місці. можлиов коли зроблять хедер и герой воно стане на місці. Бдем дивитися

gr18teamlead reviewed (changes requested):
По веикому рахунку все добре. Декілька правок вніс. 
Єдине - передивитися назви класів. А також попрошу звернути увагу на кнопку яка є вже окремим sass файлом і використати її

Sava12012 committed:
Внесено вказані правки

Не вніс тільки правку стовсовно переробки коду по бему, так як при внесенні змін все чомусь ламається ;(

gr18teamlead commented on src/sass/components/_mobile-menu.scss:
```diff
@@ -17,6 +17,11 @@
 
+  @media screen and (min-width: 768px) {
+    padding-top: 58px;
+    padding-bottom: 704px;
```
в мене по макету 690px

gr18teamlead commented on src/sass/components/_mobile-menu.scss:
```diff
@@ -41,8 +46,11 @@
 
-  padding-top: 13px;
-  padding-bottom: 13px;
+  height: 36px;
```
по макету 42px

gr18teamlead reviewed (changes requested):
Ще трохи помилок. І кнопка закриття все ще трохи менша ніж та, зо на макеті

Sava12012 committed:
Update _mobile-menu.scss

Sava12012 committed:
Update _mobile-menu.scss

gr18teamlead committed:
Merge branch 'main' into mobile-menu
